io/disk/disktest.py

```diff
@@ -0,0 +1,115 @@
+import os
```

Comment:
Hello @Nosheen08, please include the license on top of this file. You can inspire here https://github.com/avocado-framework/avocado-misc-tests/pull/26/commits/309eb378728d0fe3bfd2d0573967341a8bb07b59